app/config/processing.js

```diff
@@ -0,0 +1,29 @@
+const Joi = require('joi')
+
+// Define config schema
+const schema = Joi.object({
+  pollingInterval: Joi.number().default(10000), // 10 seconds
+  maxProcessingTries: Joi.number().default(3),
+  useEvents: Joi.boolean().default(true),
+  pollingActive: Joi.boolean().default(true)
+})
+
+// Build config
+const config = {
+  pollingInterval: process.env.POLLING_INTERVAL,
+  maxProcessingTries: process.env.MAX_PROCESSING_TRIES,
+  useEvents: process.env.USE_V2_EVENTS,
```

Comment:
Suggest renaming the environment variable to `USE_EVENTS` and drop the V2 part.  Will need reflecting in the Helm chart too